Main/Peripherals/src/Peripherals/CommandDispatcher.cpp

```diff
@@ -4,19 +4,24 @@
 
 #include "CommandDispatcher.h"
 
-CommandDispatcher::CommandDispatcher(std::istream& in, ThrustController& thrustController, PowerManager& powerManager, IHeadlights& lights)
-        : in_(in), thrustController_(thrustController), powerManager_(powerManager), lights_(lights), shouldExit_(false) {}
+CommandDispatcher::CommandDispatcher(ThrustController& thrustController, PowerManager& powerManager, IHeadlights& lights)
+        : thrustController_(thrustController), powerManager_(powerManager), lights_(lights), shouldExit_(false) {}
 
-void CommandDispatcher::runLoop() {
+void CommandDispatcher::runLoop(std::istream& in) {
```

Comment:
Passing this in as a parameter to this function instead of saving it as a member variable is a nice change. I approve.